apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -1,99 +1,76 @@
-
-Read replicas have many applications:
-
-- They reduce read load on the primary database.
-  - For example, you can use a read replica for complex analytical queries and reserve the primary for user-facing create, update, and delete operations.
-- For projects that need to cover a global audience, additional databases can be spun up closer to users in any of our twelve supported regions to reduce latency.
-- Read replicas provide data redundancy.
-
-## How are Read Replicas made?
-
-We use a hybrid approach to replicate data from a primary to its read replicas, combining the native methods of streaming replication and file-based log shipping.
-
-### Streaming replication
-
-Postgres generates a Write Ahead Log (WAL) as database changes occur. With streaming replication, these changes stream from the primary to the read replica server. The WAL alone is sufficient to reconstruct the database to its current state.
-
-This replication method is fast, since changes are streamed directly from the primary to the read replica. On the other hand, it faces challenges when the read replica can't keep up with the WAL changes from its primary. This can happen when the read replica is too small, running on degraded hardware, or has a heavier workload running.
-
-To address this, Postgres does provide tunable configuration, like `wal_keep_size`, to adjust the WAL retained by the primary. If the read replica fails to “catch up” before the WAL surpasses the `wal_keep_size` setting, the replication is terminated. Tuning is a bit of an art - the amount of WAL required is variable for every situation.
-
-### File-based log shipping
-
-In this replication method, the primary continuously buffers WAL changes to a local file and then sends the file to the read replica. If multiple read replicas are present, files could also be sent to an intermediary location accessible by all. The read replica then reads the WAL files and applies those changes. There is higher replication lag than streaming replication since the primary buffers the changes locally first. It also means there is a small chance that WAL changes do not reach read replicas if the primary goes down before the file is transferred. In these cases, if the primary fails a replica using streaming replication would (in most cases) be more up-to-date than a replica using file-based log shipping.
-
-### File-based log shipping 🤝 streaming replication
-
 <figure className="max-w-[700px] mx-auto">
     <Image
         className="hidden dark:block"
-        alt="How replication is achieved."
-        src="/docs/img/guides/platform/read-replicas/streaming-replication-dark.png?v=1"
+        alt="Map view of all project databases"
```

Comment:
no point in doing the double image if light and dark mode images are the same, you can just put in a single image and remove the className